[2025-06-07 06:32:06] ERROR: ‚ùå Error during cron job: AxiosError: Request failed with status code 500
    at settle (/Users/sabin/Desktop/My Projects/Freelance/ai-assistant-backend/node_modules/axios/dist/node/axios.cjs:2049:12)
    at IncomingMessage.handleStreamEnd (/Users/sabin/Desktop/My Projects/Freelance/ai-assistant-backend/node_modules/axios/dist/node/axios.cjs:3166:11)
    at IncomingMessage.emit (node:events:530:35)
    at endReadableNT (node:internal/streams/readable:1698:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
    at Axios.request (/Users/sabin/Desktop/My Projects/Freelance/ai-assistant-backend/node_modules/axios/dist/node/axios.cjs:4276:41)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async handleTaskAgentCronJob (/Users/sabin/Desktop/My Projects/Freelance/ai-assistant-backend/cronJob/taskAgentJob.js:144:42)
    at async Task._execution (/Users/sabin/Desktop/My Projects/Freelance/ai-assistant-backend/index.js:226:9)ss
